exercise/templatetags/exercise.py

```diff
@@ -213,14 +213,14 @@ def min_group_size(context):
 
 @register.simple_tag(takes_context=True)
 def module_accessible(context, entry):
+    t = entry.get('reading_opening_time', entry.get('opening_time'))
+    return is_accessible(context, entry, t)
+
+
+@register.simple_tag(takes_context=True)
+def exercise_accessible(context, entry):
     t = entry.get('opening_time')
-    if t and t > _prepare_now(context):
-        return False
-    if entry.get('requirements'):
-        points = _prepare_context(context)
-        module = CourseModule.objects.get(id=entry['id'])
-        return module.are_requirements_passed(points)
-    return True
+    return is_accessible(context, entry, t)
```

Comment:
It is confusing that `context` and `entry` are given to the `is_accessible` function call in a different order than in the definition of the function.

Reply:
Looks like that it is OK to switch the order of the two variables in the definition of the `is_accessible` function. The `context` variable has been the first one in other functions too.

Reply:
Switched parameters in the definition of `is_accessible` function.